tools/wheels/cibw_before_build_linux.sh

```diff
@@ -1,13 +1,22 @@
-set -xe
+NIGHTLY_FLAG=""
+
+if [ "$#" -eq 1 ]; then
```

Comment:
If we're starting to create command line options to a bash script is it worth using [getopt](https://www.baeldung.com/linux/bash-parse-command-line-arguments)? If --nightly and . are swapped around then the script won't work as intended.